src/main/java/org/reaktivity/specification/socks/internal/SocksFunctions.java

```diff
@@ -89,9 +108,22 @@ public SocksBeginExBuilder typeId(
         }
 
         public SocksBeginExBuilder address(
-            String address)
+            String address) throws UnknownHostException
+        {
+            final InetAddress inet = InetAddress.getByName(address);
+            final byte[] ip = inet.getAddress();
+            final Consumer<Builder> addressBuilder = inet instanceof Inet4Address ?
+                b -> b.ipv4Address(s -> s.put(ip)):
+                b -> b.ipv6Address(s -> s.put(ip));
```

Comment:
Please hoist duplicated code from lines `113-117` and `60-64` into a static helper method instead.

Reply:
done.